.docker/Dockerfile.test

```diff
@@ -10,25 +10,28 @@ FROM govcms8lagoon/test:${GOVCMS_IMAGE_VERSION}
 
 ENV WEBROOT=web
 
+# @todo find out why this.
 RUN mv /app/tests /tests
+
 RUN rm -rf /app
 COPY --from=cli /app /app
 
 # Copy the test scripts.
-COPY .docker/scripts/test/behat /usr/bin/behat
-COPY .docker/scripts/test/phpunit /usr/bin/phpunit
-COPY .docker/scripts/test/drutiny /usr/bin/drutiny
-COPY .docker/scripts/test/lint /usr/bin/lint
-
-RUN chmod +x /usr/bin/behat
-RUN chmod +x /usr/bin/phpunit
-RUN chmod +x /usr/bin/drutiny
-RUN chmod +x /usr/bin/lint
-
-# Note: This is currently disabled until parallel drutiny work is resolved.
-# RUN php -d memory_limit=-1 /usr/local/bin/composer --working-dir=/app/web/sites/all/drutiny/ install --ignore-platform-reqs \
-# && rm -Rf /app/web/sites/all/drutiny/.git \
-# && chmod +x /usr/bin/drutiny
-
-# Since `cli` ran --no-dev, run composer install again to get all dev tools.
-RUN composer install
+RUN cp /app/vendor/govcms/scaffold-tooling/scripts/govcms-deploy /usr/local/bin/
+RUN cp /app/vendor/govcms/scaffold-tooling/scripts/tests/* /usr/local/bin/
+RUN chmod +x /usr/local/bin/*
+
+# Install govcms/site-audit outside of the mount, although it needs to go into the mount at test time.
+# @todo audit-site/drutiny should move to the test container. Not doing any optimisation or cleanup here.
+WORKDIR /opt
+RUN git clone --branch 7.x-3.x --depth 1 https://github.com/govCMS/audit-site.git
+RUN composer --working-dir=audit-site update --ignore-platform-reqs
+RUN chmod 775 audit-site/vendor/bin/drutiny
+
+WORKDIR /app
+
+# Make drush-launcher default, sorta per https://github.com/amazeeio/lagoon/pull/1183
```

Comment:
can you please remove `sorta` and provide more explanation why this is a temporary solution

Reply:
This is because it needs socialising among the team about whether drush-launcher can be set in all upstream containers as default over drush 8, which may have implications on the nightly drutiiny process. The immediate follow up would be to fix govcms8lagoon images so they have launcher in the same way Michael requests in the Lagoon issue.

Reply:
so, why don't you say just that?!

Reply:
Because i'm in progress of pulling together code from 4 PoCs over 6 weeks so some of this I've not touched recently and the comments are targeted at me :)